src/generated/rpg_saveeasyrpgdata.h

```diff
@@ -0,0 +1,45 @@
+/* !!!! GENERATED FILE - DO NOT EDIT !!!!
+ * --------------------------------------
+ *
+ * This file is part of liblcf. Copyright (c) 2018 liblcf authors.
+ * https://github.com/EasyRPG/liblcf - https://easyrpg.org
+ *
+ * liblcf is Free/Libre Open Source Software, released under the MIT License.
+ * For the full copyright and license information, please view the COPYING
+ * file that was distributed with this source code.
+ */
+
+#ifndef LCF_RPG_SAVEEASYRPGDATA_H
+#define LCF_RPG_SAVEEASYRPGDATA_H
+
+// Headers
+#include "enum_tags.h"
+
+/**
+ * RPG::SaveEasyRpgData class.
+ */
+namespace RPG {
+	class SaveEasyRpgData {
+	public:
+		enum EasyRpg_Version {
+			EasyRpg_Version_legacy = 0,
+			EasyRpg_Version_060 = 1
```

Comment:
I'm not sure this is the right place to encode this mapping. Essentially we're creating a dependency on Player versions from liblcf.

I would probably just make it an int at the liblcf level, with a constant for the 0 legacy value. The mapping of chunk integer values to easyrpg versions could be done in Player.

Reply:
makes sense.
But that PrepareSave sets the version field to 1 is okay?

Reply:
Lets add a version argument to prepare save that gets passed in by player. Hardcoding 1 into that method is basically no different than this enum.

Reply:
Also, the value we put in could just directly map to the release version:
0.6.0 -> 600
0.10.12 -> 1012
1.0 -> 10000

Then its pretty obvious from inspection and we don't need an extra mapping.